src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier extends ColorSupplier {
+    private static final int FIGURE_COUNT = 5;
+    private static final String DEFAULT_COLOR = Color.WHITE.name();
+    private static final int DEFAULT_RADIUS = 10;
+    private static int getRandomInt = new Random().nextInt(100);
+    private static double getRandomDouble = new Random().nextDouble();
```

Comment:
These will calculate int and double one time when this class is loaded. We need new random value each time.
We need to have `private Random random` field.